eval.py

```diff
@@ -117,14 +117,15 @@ def batched_inference(models, embeddings,
         rays = sample['rays'].cuda()
         results = batched_inference(models, embeddings, rays,
                                     args.N_samples, args.N_importance, args.use_disp,
-                                    args.chunk,
-                                    dataset.white_back)
+                                    args.chunk)
+        typ = 'fine' if 'rgb_fine' in results else 'coarse'
+
+        img_pred = results[f'rgb_{typ}'].view(h, w, 3).cpu().numpy()
 
-        img_pred = results['rgb_fine'].view(h, w, 3).cpu().numpy()
-        
         if args.save_depth:
-            depth_pred = results['depth_fine'].view(h, w).cpu().numpy()
-            depth_pred = np.nan_to_num(depth_pred)
+            depth_pred = results[f'depth_{typ}'].view(h, w).cpu().numpy()
+            depth_pred_ = (depth_pred * 255).astype(np.uint8)
```

Comment:
Have you verified that this generates visually-plausible (and consistent) depth images? From my first sight no, because `depth_pred` is depth value, it can be anything like 1 or 10 or 3.1415926. Simply multiplying by 255 will not do the trick. You can refer to my `utils/visualization.py` to see how I draw depth images in the tensorboard log, but I'm not sure this generates visually-consistent depth images across the whole video (probably not). You need to sort of fix a minimum and maximum depth value across the whole video and normalize every depth according to them, then finally draw the depth images.

Reply:
good point, will fix